src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = new Lottery[NUMBER_OF_BALLS];
+
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = new Lottery();
+            System.out.println(balls[i].getRandomBall());
+        }
```

Comment:
how about this: let's use NUMBER_OF_BALLS constant in for loop condition directly, and don;t create an array at all, just call new Lotter().getRandomBall()